### DISCUSSION TITLE
problem connecting to remote

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#656] [General]
Hi all,   I recently updated to ubuntu 24.04.   Since the update my remote will not connect and show in a web document.  Here is the error message I'm getting.   Any assistance is appreciated.  The IP below is my home network Windows PC. I'm running the latest Slinger.

ConnectionRefusedError: [Errno 111] Connection refused
06/08/2024, 14:36:28.001   RemoteControl connection from ('192.168.55.142', 63948)
Exception in thread Thread-25 (remote_control_stream):
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1073, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1010, in run
    self._target(*self._args, **self._kwargs)
  File "/home/slingbox/slingbox_server.py", line 987, in remote_control_stream
    remote_control_socket.connect(('127.0.0.1', http_port ))  ## Send Packets to Flask

**Comment by GerryDazoo** [comment]
My first guess is it's a firewall issue but usually the default setting allow all traffic on the loopback interface 127.0.0.1/localhost.

**Comment by pfuller88** [comment]
Thanks.  I keep troubleshooting.
